### DISCUSSION TITLE
Github Badges

### DISCUSSION THREAD

**Discussion by Peter0298** [community/community#130973] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

Help me set up my new GitHub profile and earn badges by suggesting some beginner-friendly projects or tutorials to get me started!

**Comment by karol767** [comment, marked as answer]
If you want awesome github profile, reference this urls.

https://github.com/github-profile-achievements/english
https://github.com/orgs/community/discussions/121833
https://github.com/anuraghazra/github-readme-stats/edit/master/readme.md
https://www.linkedin.com/pulse/create-cool-github-profile-readmemd-muhammedh-shadir
https://www.sitepoint.com/github-profile-readme/

**Reply by Monireh1978** [reply]
hi i nee ur help
ru fur ETH wallet

**Reply by Monireh1978** [reply]
ineed a peson to help me to my account in ETH return

**Reply by Monireh1978** [reply]
0x5e7253922AF18Ca8EE4490A14Eb21b6BbA294433
ist that hier

**Comment by cloudsetin** [comment]
This is really helpful for me.

**Comment by ghostinhershell** [comment]
Thanks so much for posting in our Community @Peter0298. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
